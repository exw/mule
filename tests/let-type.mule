let type t = {} in {} : t
