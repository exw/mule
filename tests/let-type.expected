
Desugared:

	(let-type ((t (rec t (record (types ()) (values ()))))) ((: t) {}))

inferred type:

	(record (types ()) (values ()))

Runtime term:

	(Record ())

Evaluated:

	(Record ())
